package.json

```diff
@@ -9,9 +9,9 @@
     "web": "expo start --web"
   },
   "dependencies": {
```

Comment:
It could be a good idea to add a scripts section of package.json to make it easier to remember, like `"prettier": "prettier . --write"`

Otherwise, looks fine to me.